README.md

```diff
@@ -1135,6 +1143,141 @@ the opposite order:
 }
 ```
 
+## SQL Support
+The following table outlines the subset of the GraphQL compiler features that are tested and
+supported for various relational database flavors:
+
+     
+|  Dialect   | Supported Directives                   | Caveats                                          |
+| ---------  | -------------------------------------- | -----------------------------------------------  |
+| PostgreSQL | Root-only queries (no edges), [@output](#output), [@filter](#filter) | [\__typename](#__typename) output metafield, [intersects](#intersects) filter,  [has_edge_degree](#has_edge_degree) filter and [name_or_alias](#name_or_alias) filter unsupported|
+| Sql Server | Root-only queries (no edges), [@output](#output), [@filter](#filter) | [\__typename](#__typename) output metafield, [intersects](#intersects) filter,  [has_edge_degree](#has_edge_degree) filter and [name_or_alias](#name_or_alias) filter unsupported|
+| MySQL      | Root-only queries (no edges), [@output](#output), [@filter](#filter) | [\__typename](#__typename) output metafield, [intersects](#intersects) filter,  [has_edge_degree](#has_edge_degree) filter and [name_or_alias](#name_or_alias) filter unsupported|
+| MariaDB    | Root-only queries (no edges), [@output](#output), [@filter](#filter) | [\__typename](#__typename) output metafield, [intersects](#intersects) filter,  [has_edge_degree](#has_edge_degree) filter and [name_or_alias](#name_or_alias) filter unsupported|
+| SQLite     | Root-only queries (no edges), [@output](#output), [@filter](#filter) | [\__typename](#__typename) output metafield, [intersects](#intersects) filter,  [has_edge_degree](#has_edge_degree) filter and [name_or_alias](#name_or_alias) filter unsupported|
+
+
+### Configuring SQLAlchemy
+Support for relational databases is accomplished by compiling to SQLAlchemy core as an intermediate
+language, and then relying on SQLAlchemy's compilation of the dialect specific SQL string to query
+the target database.
+
+For the SQL backend, GraphQL types are assumed to have a SQL table of the same name, and with the
+same properties. For example, a schema type 
+```
+type Animal {
+    name: String
+}
+```
+is expected to correspond to a SQLAlchemy table object of the same name (case-insensitive) like
```

Comment:
Can you say more about what happens if the schema does not correspond to a SQLAlchemy table? e.g. what kind of validation is in place, and what kind of result would the user get back?